node/clients/sellers.ts

```diff
@@ -84,3 +45,29 @@ interface SalesChannel {
   id: number
   name: string
 }
+
+export class SellersClient extends ExternalClient {
+  constructor(context: IOContext, options?: InstanceOptions) {
+    super(`http://${context.account}.vtexcommercestable.com.br`, context, {
+      ...options,
+      headers: {
+        ...options?.headers,
+        VtexIdClientAutCookie: context.authToken,
+      },
+    })
+  }
+
+  public sellers = async () => {
+    const sellers = await this.http.get<SellersResponse>(SELLERS_URL, {
+      metric: 'adyen-getSellers',
+    })
+
+    return sellers.items
```

Comment:
Please check the other clients to always return the whole HTTP response

Reply:
Done